roles/node_images_ncn/files/cloud/templates/cloud-init-network.mlx.tmpl

```diff
@@ -0,0 +1,31 @@
+## template:jinja
+network:
+  version: 2
+  ethernets:
+    mgmt0:
+      dhcp6: no
+      mtu: 9000
+    mgmt1:
+      dhcp6: no
+      mtu: 9000
+  bonds:
+    bond0:
+      interfaces: [mgmt0, mgmt1]
+      dhcp4: false
+      mtu: 9000
+      parameters:
+        mode: 802.3ad
+        mii-monitor-interval: 100
+        lacp-rate: fast
+        ad-select: bandwidth
+        transmit-hash-policy: encap3+4
```

Comment:
If this is just for a workaround that's fine. Encap 3+4 is not LACP compliant so this could be a risk and cause other issues.

Reply:
https://github.com/Cray-HPE/metal-provision/pull/473#discussion_r1326175206